resources/javaTemplates/BaseModelObjectTemplate.txt

```diff
@@ -123,7 +123,7 @@ public abstract class ModelObjectV3 extends CoreModelObject {
 	@Override
 	protected boolean isNoAssertion(Object value) {
 		return value instanceof IndividualLicensingInfo && 
-				((IndividualLicensingInfo)value).getObjectUri() == "https://spdx.org/rdf/{{{versionSemVer}}}/terms/Licensing/NoAssertion";
+				"https://spdx.org/rdf/3.0.1/terms/Licensing/NoAssertion".equals(((IndividualLicensingInfo) value).getObjectUri());
```

Comment:
```suggestion
				"https://spdx.org/rdf/{{{versionSemVer}}}/terms/Licensing/NoAssertion".equals(((IndividualLicensingInfo) value).getObjectUri());
```

We need to refer to the version specific URI

Reply:
Sorry, fixed.